### DISCUSSION TITLE
is it possible to mix transform css on config with existing transform styles ?

### DISCUSSION THREAD

**Discussion by branlok** [pmndrs/react-spring#1557] [Support] [answered]
I'm trying to combine two transform properties i.e. rotate and scale together. However my rotate is declared in the transition array because i need to arbitrarily generate a degree to persist, While the scale is decalared in the ```from ,enter, leave``` part of the hook. Is there any way to join the two

I have the code on here,
https://codesandbox.io/s/demo-card-x727n?file=/src/App.js

I'm trying to get these two animation to occur simultaneously
https://gyazo.com/b192d66afb5a27fdc9a62a6a269e988d 
and 

https://gyazo.com/3cdc7bcec0b149c74775ee4ee2732d40

thank you!

**Comment by branlok** [comment]
I assume, a quick patch would be adding a wrapper on top of the targeted item and add styles accordingly. but interested to know if theres a better alternative.

**Comment by joshuaellis** [comment, marked as answer]
You mean like this? https://codesandbox.io/s/demo-card-forked-hthrh?file=/src/App.js

**Reply by branlok** [reply]
oh thats great ! lol i forgot i can call the exact type of transformations. Thanks Josh
